napari/utils/action_manager.py

```diff
@@ -37,6 +37,7 @@ class Action:
     command: Callable
     description: str
     keymapprovider: KeymapProvider  # subclassclass or instance of a subclass
+    is_toggle: bool  # subclassclass or instance of a subclass
```

Comment:
```suggestion
    is_toggle_on_long_hold: bool
```

As there are only togglable actions, this flag should inform about these actions when short click change attribute, but when long hold then it restores previous value.

Reply:
Renamed :)